1 HW_Standard/HW_Cranidos.yaml

```diff
@@ -0,0 +1,22 @@
+species: Cranidos
+ingamename: Dino Badge
+displayname: Dino Badge
+setname: Halloween
+tags: [halloween]
+item: Skull Fossil
+ability: Tangled Feet
+moves:
+    - [Outrage]
```

Comment:
Outrage is too strong, the other options too weak, I would switch to modest nature, remove the bulk and max spatk instead.